src/GEMItem.h

```diff
@@ -112,6 +113,24 @@ class GEMItem {
     GEMItem(char* title_, boolean& linkedVariable_, boolean readonly_ = false);
     GEMItem(char* title_, float& linkedVariable_, boolean readonly_ = false);
     GEMItem(char* title_, double& linkedVariable_, boolean readonly_ = false);
+    /* 
+      Constructors for menu item that represents variable, that are linked to another screen
+      @param 'title_' - title of the menu item displayed on the screen
+      @param 'linkedVariable_' - reference to variable that menu item is associated with (either byte, int, char*, boolean, float, or double)
+      @param 'linkedPage_' - reference to GEMPage menu page that menu item is associated with
+      */
+    GEMItem(char* title_, byte& linkedVariable_, GEMPage* linkedPage_);
+    GEMItem(char* title_, byte& linkedVariable_, GEMPage& linkedPage_);
+    GEMItem(char* title_, int& linkedVariable_, GEMPage* linkedPage_);
+    GEMItem(char* title_, int& linkedVariable_, GEMPage& linkedPage_);
+    GEMItem(char* title_, char* linkedVariable_, GEMPage* linkedPage_);
+    GEMItem(char* title_, char* linkedVariable_, GEMPage& linkedPage_);
+    GEMItem(char* title_, boolean& linkedVariable_, GEMPage* linkedPage_);
```

Comment:
Here you add boolean.. with a previous merge request we replaced `boolean` with `bool`. It would be nice to use `bool` in new added code lines

Reply:
I fixed it